x-pack/platform/plugins/private/transform/server/lib/alerting/transform_health_rule_type/transform_health_service.ts

```diff
@@ -347,8 +360,10 @@ export function transformHealthServiceProvider({
       if (testsConfig.healthCheck.enabled) {
         const response = await this.getUnhealthyTransformsReport(transformIds);
         const isHealthy = response.length === 0;
-        const count = response.length;
-        const transformsString = response.map((t) => t.transform_id).join(', ');
+        const count: number = isHealthy ? previousState?.unhealthy?.length ?? 0 : response.length;
+        const transformsString = isHealthy
```

Comment:
As discussed, what about limiting the number of IDs we display in the first part of the message to say 10 (?), to say something like

```
35 transforms are not started: endpoint.metadata_current-default-8.18.0-prerelease.1, endpoint.metadata_united-default-8.18.0-prerelease.1, fq_continuous_latest_transform, fq_response_times_continuous, gallery_latest_transform, transform_a, transform_b, transform_c, transform_d, transform_e and 25 others
```

If a user really wants to see all the IDs they can loop through each of the IDs usinh the context variables.

Reply:
Updated in f36309a960505260a1283cf72b61f6a4348e4743